.github/workflows/continuous-integration.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           restore-keys: "${{ matrix.php-version }}-composer-locked-"
 
       - name: "Install dependencies with composer"
-        run: "composer update --no-interaction --no-progress --no-suggest"
+        run: "composer install --no-interaction --no-progress --no-suggest"
```

Comment:
Should be fixed up with previous commit

Reply:
done, but will removed once your PR is merged.